docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1059,7 +1058,30 @@ BESU_PRIVACY_URL=http://127.0.0.1:8888
 privacy-url="http://127.0.0.1:8888"
 ```
 
-URL on which the [Orion node](../../Tutorials/Privacy/Configuring-Privacy.md#4-create-orion-configuration-files) is running.    
+URL on which the [Orion node](../../Tutorials/Privacy/Configuring-Privacy.md#4-create-orion-configuration-files) is running.
+
+### pruning-enabled
+
+```bash tab="Syntax"
+--pruning-enabled
+```
+
+```bash tab="Command Line"
+--pruning-enabled=true
+```
+
+```bash tab="Environment Variable"
+BESU_PRUNING_ENABLED=true
+```
+
+```bash tab="Configuration File"
+pruning-enabled=true
+```
+
+Enables pruning of unused world state of old blocks that will not be required except by chain reorganizations. 
```

Comment:
This relates a bit to my first comment on the conceptual material.  If I didn't already know what pruning was, I could read this and still be wondering why I would enable pruning.  

It won't work until the pruning concept material is added but it's probably easiest just to add the link now - they'll be merged at similar times.

Reply:
Updated.

Reply:
It's very important to note that the blocks are unlikely to required except by deep chain reorganizations. I also think it's not something that needs to be mentioned in the one-liner for what the option does. But definitely in the conceptual overview of pruning.

Reply:
There was some discrepancy around this comment because the "outdated" label was missed, which became evident in a Slack discussion. I added the line about the pruned state trie nodes being not required to the [PR for the concept topic](https://github.com/hyperledger/besu-docs/pull/66) and changed chain to world state here.